Exercise_C.js

```diff
@@ -15,4 +15,13 @@ See README.md if you need help running your code.
 
 for (let i = 1; i <= 30; i++) {
   // CODE HERE
+  if (i % 15 === 0) {
+    console.log('Hong Kong');
+  } else if (i % 3 === 0) {
+    console.log('Hong');
+  } else if (i % 5 === 0) {
+    console.log('Kong');
+  } else {
+    console.log(i);
+  }
 }

```

Comment:
Great Work with this Exercise, good use of using the number 15 as a common number between 3 and 5.